src/lucky_web/tags/base_tags.cr

```diff
@@ -35,7 +36,11 @@ module LuckyWeb::BaseTags
   end
 
   {% for tag in TAGS %}
-    generate_tag_methods({{tag.id}})
+    generate_tag_methods({{tag.id}}, {{tag.id}})
+  {% end %}
+
+  {% for name, tag in RENAMED_TAGS %}
+    generate_tag_methods({{name}}, {{tag}})
```

Comment:
Could you use named args here and above so it's a bit more clear

```crystal
generate_tag_methods(method_name: {{ name }}, tag: {{ tag }})
```

Reply:
That's a lot better, good call!